tests/test_config.py

```diff
@@ -46,6 +47,13 @@ def test_config_from_json():
     common_object_test(app)
 
 
+def test_config_from_file():
```

Comment:
Rename to `test_config_from_toml_file`.

Reply:
I've removed it and added some docs instead.